compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -99,10 +99,10 @@ void addPadBasedPassPipeline(OpPassManager &pm, TilingConfig &tilingConfig) {
     }
     if (i == 2) {
       modulePassManager.addNestedPass<func::FuncOp>(createAMDAIECleanupPass());
-      pm.addPass(createCanonicalizerPass());
-      pm.addPass(createCSEPass());
     }
     {
+      modulePassManager.addNestedPass<func::FuncOp>(createCanonicalizerPass());
+      modulePassManager.addNestedPass<func::FuncOp>(createCSEPass());
```

Comment:
If they aren't being guarded then they can just be pulled out of the `{....}` block.